### PR TITLE
Add tracking endpoint

### PR DESCRIPTION
**Pull request by webmasterjunkie**
Also changes the campaign list endpoint to include the optional status
parameter.

### CODE REVIEW

File: src/apiary.apib [hunk 1]
```diff
@@ -879,10 +879,14 @@ Before a Publishers can begin to generate conversions, they must be associated w
 endpoint allows the Publisher to request access to a Campaign, and to agree to any Terms and Conditions
 that the Campaign defines.
 
-## Publisher Campaigns [/user/publisher/{publisher_id}/campaign]
+## Publisher Campaigns [/user/publisher/{publisher_id}/campaign/{status}]
 
 + Parameters
     + publisher_id: 1l1007802 - ID of the publisher
+    + status: (string, optional) - The status of the publisher on the campaign
+        + a - approved
```
Comment (pete001) [inline src/apiary.apib:887]: This should be a data structure, enum.

File: src/apiary.apib [hunk 2]
```diff
@@ -892,6 +896,23 @@ View all Campaigns that the Publisher is associated with, along with the status
     + Attributes
         + campaigns (array[Publisher Campaign Wrapper])
 
+## Publisher Campaign Tracking [/user/publisher/{publisher_id}/campaign/{status}/tracking]
+
++ Parameters
+    + publisher_id: 1l1007802 - ID of the publisher
+    + status: (string) - The status of the publisher on the campaign
+        + a - approved
```
Comment (pete001) [inline src/apiary.apib:904]: It looks like there the following in common.apib: 

```
## Status (enum)
+ a - Approved
+ p - Pending
+ r - Rejected
+ n
```

Comment (pete001) [inline src/apiary.apib:904]: If `n` isnt valid, then we you could create a new status for a/p/r and `Include` that into this existing Status (enum)

File: src/apiary.apib [hunk 3]
```diff
@@ -892,6 +896,23 @@ View all Campaigns that the Publisher is associated with, along with the status
     + Attributes
         + campaigns (array[Publisher Campaign Wrapper])
 
+## Publisher Campaign Tracking [/user/publisher/{publisher_id}/campaign/{status}/tracking]
+
++ Parameters
+    + publisher_id: 1l1007802 - ID of the publisher
+    + status: (enum) - The status of the publisher on the campaign
```
Comment (pete001) [inline src/apiary.apib]: This is required in this instance.

Comment (webmasterjunkie) [inline src/apiary.apib]: Correct. The docs show the absence of (required) defaults to [required](https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#description-13). I have added the data type for specifics now.

**Pull request merged**